edgelet/hsm-sys/azure-iot-hsm-c/src/edge_pki_openssl.c

```diff
@@ -871,19 +872,47 @@ static int set_key_usage
     return result;
 }
 
+static int set_san
+(
+    X509 *x509_cert,
+    CERT_PROPS_HANDLE cert_props_handle
+)
+{
+    size_t num_entries = 0, idx;
+    bool fail_flag = false;
+    const char * const* sans = get_san_entries(cert_props_handle, &num_entries);
```

Comment:
Curious - where is this defined? Was that this the last PR?

Reply:
Yes that is correct. I just wanted to simplify reviews.